### PR TITLE
updating app.js to fit with the modification in the back for the disp…

### PR DESCRIPTION
**Pull request by Dezedene**
Updating app.js to fit with the modification in the back for the display of the sample pois at loading.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,12 +20,16 @@ class App extends Component {
...
-      .catch(err => console.log(err));
-    axios.get('http://localhost:3001/pois/keywords')
-      .then(response => dispatch({ type: 'GET_POIS_KEYWORDS', poiKeywordsDisplay: response.data }))
-      .catch(err => console.log(err));
+  }
+
+  componentDidUpdate(prevProps) {
+    const { dispatch, geolocCoordonnees } = this.props;
+    if (geolocCoordonnees !== prevProps.geolocCoordonnees) {
+      console.log(geolocCoordonnees);
```
Comment (erabin05) [inline src/App.js]: remove the console.log()

**Review by erabin05 — CHANGES_REQUESTED** [review]
- remove the console.log()

File: src/App.js (hunk 1 above)
Comment (0figueiredo) [reply src/App.js]: console.log() removed

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**